library/src/org/wordpress/passcodelock/PasscodeUnlockActivity.java

```diff
@@ -1,38 +1,69 @@
 package org.wordpress.passcodelock;
 
 import android.content.Intent;
-import android.view.animation.Animation;
-import android.view.animation.AnimationUtils;
+import android.support.v4.hardware.fingerprint.FingerprintManagerCompat;
+import android.support.v4.os.CancellationSignal;
+import android.view.View;
 
 public class PasscodeUnlockActivity extends AbstractPasscodeKeyboardActivity {
-    
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        // Hide fingerprint notification if the hardware doesn't support it
+        if (!mFingerprintManager.isHardwareDetected() ||
+            !mFingerprintManager.hasEnrolledFingerprints()) {
```

Comment:
You can leave this because this is minor, but how about setting default visibility in the xml as `gone`, and only `if (mFingerprintManager.isHardwareDetected() && mFingerprintManager.hasEnrolledFingerprints())` do we set it to visible and authenticate?